index.html

```diff
@@ -10,7 +10,71 @@
   <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
 </head>
 <body>
-  <h1 class="title">Front-End</h1>
-  <h1 class="title">Back-End</h1>
+  <div class="parent">
+    
+    <h1 class="title">Front-End Tools</h1>
+    <div class="tech-collection">
+      
+      <div class="tech">
+        <img class="icon" src="assets/react_icon.svg"/>
```

Comment:
Every `img` tag should have an `alt` attribute saying what's on the image so it can be shown in case it doesn't load :smile: